packages/core/src/dynamoDb/explorer/dynamoDbTableNode.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DynamoDB } from 'aws-sdk'
+import { AWSResourceNode } from '../../shared/treeview/nodes/awsResourceNode'
+import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
+
+export class DynamoDbTableNode extends AWSTreeNodeBase implements AWSResourceNode {
+    public constructor(public override readonly regionCode: string, public dynamoDbtable: DynamoDB.Types.TableName) {
+        super('')
+        this.update(dynamoDbtable)
+    }
+
+    public update(dynamoDbtable: DynamoDB.Types.TableName): void {
+        this.dynamoDbtable = dynamoDbtable
```

Comment:
Is there a reason this is in a separate function? I think if you moved this into the constructor you could get rid of the `!` operators on lines 23 and 27. `!` operators are scary in typescript because it asserts that the variable is 100% not undefined or null, which may or may not actually be the case at runtime

Reply:
There is no specific reason, I have moved it to constructor.